common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -148,6 +148,10 @@ div.problem {
       margin-top: $baseline;
     }
   }
+
+  .MathJax_Preview, .MathJax_CHTML {
```

Comment:
I agree with the @cahrens I have looked on the recent stage with latest hotfix, now MathJax_CHTML is replaced with MathJax_SVG. But @cptvitamin is right we need to cover all the above mentioned classes so that if any is used we may cover them well.

Also, I noted that now you may need to do more than just displaying inline-bock to these classes, you may need to change a bit more.